extensions/amp-base-carousel/0.1/test-e2e/helpers.js

```diff
@@ -33,7 +33,7 @@ export async function waitForCarouselImg(controller, n) {
   // container. We query all the imgs upfront, since they might not have
   // laid out yet.
   const el = await controller.findElementXPath(
-      `//${TAG_NAME}//div[contains(@class, "${SLOTTED_CLASS}")][${n + 1}]` +
+      `//${TAG_NAME}//*[contains(@class, "${SLOTTED_CLASS}")][${n + 1}]` +
```

Comment:
It looks like the wrapper `div` was removed at some point and the xpath wasn't matching the elements any more

Reply:
Oh right, I did refactor the manual test files to do that.